x-pack/filebeat/input/httpjson/response.go

```diff
@@ -74,63 +74,68 @@ func newResponseProcessor(config *responseConfig, pagination *pagination, log *l
 	return rp
 }
 
-func (rp *responseProcessor) startProcessing(stdCtx context.Context, trCtx *transformContext, resp *http.Response) (<-chan maybeMsg, error) {
+func (rp *responseProcessor) startProcessing(stdCtx context.Context, trCtx *transformContext, resp []*http.Response) (<-chan maybeMsg, error) {
```

Comment:
`resp` is a slice, it could be called `resps`
```suggestion
func (rp *responseProcessor) startProcessing(stdCtx context.Context, trCtx *transformContext, resps []*http.Response) (<-chan maybeMsg, error) {
```